torch/library.py

```diff
@@ -491,34 +515,23 @@ def register_fake(qualname, func=None, /, *, lib=None, _stacklevel=1):
         >>> assert torch.allclose(trace(x), torch.ops.mylib.custom_nonzero(x))
 
     """
-    source = torch._library.utils.get_source(_stacklevel + 1)
-    frame = sys._getframe(_stacklevel)
-    caller_module = inspect.getmodule(frame)
-    # Can be none if you call register_fake from somewhere there isn't a module
-    # (e.g. __main__)
-    caller_module_name = None if caller_module is None else caller_module.__name__
-
-    # TODO(rzou): We're gonna need to stage this change with torchvision,
-    # since torchvision is github first.
-    if caller_module_name is not None and caller_module_name.startswith("torchvision."):
-        caller_module_name = None
-
-    def inner(func):
-        entry = torch._library.simple_registry.singleton.find(qualname)
-        if caller_module_name is not None:
-            func_to_register = _check_pystubs_once(func, qualname, caller_module_name)
-        else:
-            func_to_register = func
+    stacklevel = _stacklevel
```

Comment:
Why this rename here and not in the arg?
Private args in python are a lie anyways, everyone can see them haha

Reply:
I needed to update the stacklevel below and didn't want to modify the `_stacklevel` argument